doc/ssl/SSL_SESSION_get_time.pod

```diff
@@ -4,6 +4,8 @@
 
 SSL_SESSION_get_time, SSL_SESSION_set_time, SSL_SESSION_get_timeout, SSL_SESSION_set_timeout - retrieve and manipulate session time and timeout settings
 
+SSL_SESSION_get0_hostname - retrieve the hostname for the session
```

Comment:
The NAME section is in a very strict format. It is actually parsed by a perl script during the man page generation. This change will break that. Remove this line and add the function name to the list of functions above. You probably should also tweak the description to say something like "retrieve and manipulate session time, timeout and hostname settings"